src/modules/m_bot.c

```diff
@@ -549,8 +549,8 @@ void bot_rkb (Nick *nptr, User *uptr, Chan *chptr, Wchan *wchan, char *all)
         LLIST_FOREACH_ENTRY_SAFE(&wchan->members, member, tmp_member, wchan_head) {
             if (member->nick == nptr)
                 continue;
-            if (__match_regex(arg2, member->nick->nick, REG_EXTENDED|REG_NOSUB|REG_ICASE)) {
-                uptr2 = find_user(member->nick->nick);
+            if (__match_regex(arg2, member->nick->svid, REG_EXTENDED|REG_NOSUB|REG_ICASE)) {
```

Comment:
In this case, we are actually targeting nicks and not accounts, as we want to kickban members from a channel. Using find_account on the line below is fine, however the regexp match should be performed on the nick rather than the svid.

Reply:
Fixed